app/cdap/components/GridTable/index.tsx

```diff
@@ -36,11 +36,15 @@ import { useStyles } from 'components/GridTable/styles';
 import {
   IExecuteAPIResponse,
   IHeaderNamesList,
-  IObject,
+  IMissingList,
   IParams,
   IRecords,
+  ICellData,
```

Comment:
I think we should think of more specific name here, instead of ICellData
plz check why it is used and rename accordingly